pyincore/utils/cgeoutputprocess.py

```diff
@@ -149,12 +150,8 @@ def get_cge_employment(pre_demand, post_demand,
             pre_disaster_demand = pre_demand.get_dataframe_from_csv()
             post_disaster_demand = post_demand.get_dataframe_from_csv()
 
-        before_values = [pre_disaster_demand["GOODS"].sum(),
-                         pre_disaster_demand["TRADE"].sum(),
-                         pre_disaster_demand["OTHER"].sum()]
-        after_values = [post_disaster_demand["GOODS"].sum(),
-                        post_disaster_demand["TRADE"].sum(),
-                        post_disaster_demand["OTHER"].sum()]
+        before_values = [pre_disaster_demand[demand_category].sum() for demand_category in demand_categories]
```

Comment:
I am still having the same error. Here's the columns that pre-disaster-factor-demand.csv 
```
Labor Group,AG_MI_R1,UTIL_R1,CONS_R1,MANU_R1,COMMER_R1,EDU_R1,HEALTH_R1,ART_ACC_R1,RELIG_R1,AG_MI_R2,UTIL_R2,CONS_R2,MANU_R2,COMMER_R2,EDU_R2,HEALTH_R2,ART_ACC_R2,RELIG_R2,AG_MI_R3,UTIL_R3,CONS_R3,MANU_R3,COMMER_R3,EDU_R3,HEALTH_R3,ART_ACC_R3,RELIG_R3,AG_MI_R4,UTIL_R4,CONS_R4,MANU_R4,COMMER_R4,EDU_R4,HEALTH_R4,ART_ACC_R4,RELIG_R4,AG_MI_R5,UTIL_R5,CONS_R5,MANU_R5,COMMER_R5,EDU_R5,HEALTH_R5,ART_ACC_R5,RELIG_R5,AG_MI_R6,UTIL_R6,CONS_R6,MANU_R6,COMMER_R6,EDU_R6,HEALTH_R6,ART_ACC_R6,RELIG_R6,AG_MI_R7,UTIL_R7,CONS_R7,MANU_R7,COMMER_R7,EDU_R7,HEALTH_R7,ART_ACC_R7,RELIG_R7,L1W_R1,L2W_R1,L3W_R1,L4W_R1,L1W_R2,L2W_R2,L3W_R2,L4W_R2,L1W_R3,L2W_R3,L3W_R3,L4W_R3,L1W_R4,L2W_R4,L3W_R4,L4W_R4,L1W_R5,L2W_R5,L3W_R5,L4W_R5,L1W_R6,L2W_R6,L3W_R6,L4W_R6,L1W_R7,L2W_R7,L3W_R7,L4W_R7,KAP,INVES,HS1_R1,HS2_R1,HS1_R2,HS2_R2,HS1_R3,HS2_R3,HS1_R4,HS2_R4,HS1_R5,HS2_R5,HS1_R6,HS2_R6,HS1_R7,HS2_R7,HH1_R1,HH2_R1,HH3_R1,HH4_R1,HH1_R2,HH2_R2,HH3_R2,HH4_R2,HH1_R3,HH2_R3,HH3_R3,HH4_R3,HH1_R4,HH2_R4,HH3_R4,HH4_R4,HH1_R5,HH2_R5,HH3_R5,HH4_R5,HH1_R6,HH2_R6,HH3_R6,HH4_R6,HH1_R7,HH2_R7,HH3_R7,HH4_R7,USSOC1_R1,USSOC2_R1,USSOC3_R1,USSOC4_R1,USSOC1_R2,USSOC2_R2,USSOC3_R2,USSOC4_R2,USSOC1_R3,USSOC2_R3,USSOC3_R3,USSOC4_R3,USSOC1_R4,USSOC2_R4,USSOC3_R4,USSOC4_R4,USSOC1_R5,USSOC2_R5,USSOC3_R5,USSOC4_R5,USSOC1_R6,USSOC2_R6,USSOC3_R6,USSOC4_R6,USSOC1_R7,USSOC2_R7,USSOC3_R7,USSOC4_R7,PROTAX,UTSTX,COUNTYSTX,UTITX,USPIT,OTX,CYGF,COUNTY,STATE,FED,OUT1,OUT2,OUT3,OUT4,ROW
```
However, the test codes are looking for "GOODS, TRADE, OTHER" and it makes an error since those are not in the table, I believe